inc/osvr/ClientKit/Imaging.h

```diff
@@ -94,29 +92,25 @@ namespace clientkit {
                                      const struct OSVR_ImagingReport *report) {
                 ImagingCallbackRegistration *self =
                     static_cast<ImagingCallbackRegistration *>(userdata);
-                ImagingReportOpenCV newReport;
+                ImagingReport newReport;
                 newReport.sensor = report->sensor;
                 newReport.buffer.reset(report->state.data,
                                        ImagingDeleter(self->m_ctx));
-                newReport.frame = cv::Mat(
```

Comment:
This is just in the optional C++ header-only wrapper. While it's recommended, because of typing reasons, it's not strictly required for use of even the imaging interface. There's not much point in having this extra layer wrapping the callbacks without marshalling the data into the OpenCV type - that's what this file exists for. If you just want the bytes, there's the C interface.